### DISCUSSION TITLE
Is it possible to set the width and/or height of a field?

### DISCUSSION THREAD

**Discussion by the1gofer** [mProjectsCode/obsidian-meta-bind-plugin#168] [Q&A] [answered]
Is it possible to set the width and/or height of a field?  In my case, specifically the Text Area field. 

It's pretty small by default, and I'd like to make it look a littl better.

**Comment by mProjectsCode** [comment, marked as answer]
Yes, with a css snippet and the `class` argument.

**Comment by the1gofer** [comment]
Ok, I don't know a ton about css.  THanks.
Jason Crews

**Reply by motion2082** [reply]
Agreed there needs to be more effort put into the documentation on how to apply custom CSS styles with examples

At the moment it's impossible to work out how to remove default styles

**Reply by mProjectsCode** [reply]
I assure you that effort is being put into the docs, but I am running a one-man show with limited time.

**Reply by motion2082** [reply]
Thank you for all your efforts, your plugin is amazing!

One of the hardest things to do is to ask for help but I’m sure many people would be willing to contribute. I see a lot of great feedback and suggestions here on GitHub.
